recipes/pstdint/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.1.16.0" %}
+
+package:
+  name: pstdint
+  version: {{ version }}
+
+source:
+  fn: pstdint-{{ version }}.h
+  url: http://www.azillionmonkeys.com/qed/pstdint.h
+  sha256: aed1262c00f5eb645354d00c491236172841cf7728115c39a1f77f305fc7db12
+
+build:
+  number: 0
+  script:
+    - mkdir -p "${PREFIX}/include"                                 # [unix]
+    - if not exist "%LIBRARY_INC%" mkdir "%LIBRARY_INC%"           # [win]
+    - cp "pstdint-{{ version }}.h" "${PREFIX}/include/pstdint.h"   # [unix]
+    - copy "pstdint-{{ version }}.h" "%LIBRARY_INC%\pstdint.h"     # [win]
+
+test:
+  commands:
+    - test -f "${PREFIX}/include/pstdint.h"                        # [unix]
+    - if not exist "%PREFIX%\\Library\\include\\pstdint.h" exit 1  # [win]
+
+about:
+  home: http://www.azillionmonkeys.com/qed/pstdint.h
+  license: BSD 3-Clause
```

Comment:
License is in the header file as a comment. Could strip it out into a separate file if needed.